ui/app/templates/components/tool-unwrap.hbs

```diff
@@ -12,41 +12,20 @@
 </PageHeader>
 
 {{#if @unwrap_data}}
-  <div class="box is-sideless is-fullwidth is-marginless">
-    <nav class="tabs has-bottom-margin-l">
-      <ul>
-        {{! template-lint-configure no-nested-interactive "warn" }}
-        <li class={{if (eq @unwrapActiveTab "data") "is-active"}}>
-          <button
-            type="button"
-            class="link link-plain tab has-text-weight-semibold"
-            {{action (mut @unwrapActiveTab) "data"}}
-            data-test-button-data
-          >
-            Data
-          </button>
-        </li>
-        <li class={{if (eq @unwrapActiveTab "details") "is-active"}}>
-          <button
-            type="button"
-            class="link link-plain tab has-text-weight-semibold"
-            {{action (mut @unwrapActiveTab) "details"}}
-            data-test-button-details
-          >
-            Wrap Details
-          </button>
-        </li>
-        {{! template-lint-configure no-nested-interactive "on" }}
-      </ul>
-    </nav>
-    {{#if (eq @unwrapActiveTab "data")}}
-      <div class="field">
-        <div class="control">
-          <JsonEditor @title="Unwrapped Data" @value={{stringify @unwrap_data}} @readOnly={{true}} />
-        </div>
-      </div>
-    {{else}}
-      <div class="field box is-fullwidth is-shadowless is-paddingless is-marginless">
+  <Hds::Tabs as |T|>
```

Comment:
<img width="1190" alt="Screenshot 2023-10-11 at 5 02 03 PM" src="https://github.com/hashicorp/vault/assets/68122737/879852a3-cdf4-4208-b8d8-30dcdbbe6bdf">

Reply:
No extra lines after the info table rows 😍